### DISCUSSION TITLE
New Switch Documentation

### DISCUSSION THREAD

**Discussion by eflann** [twilio-labs/paste#2790] [General] [answered]
Hi! 
I was working with the Switch component and noticed that the documentation is ahead of the version of paste that is actually released. 

This commit hasn't been released. 
https://github.com/twilio-labs/paste/commit/38a72ad0c64ad0e11316cf410505dcec640d0c16

The docs for paste switch are written as though this has been released. For example the prop name changed from "on" -> "checked" in the documentation.
https://paste.twilio.design/components/switch/#on

I was wondering if there were any plans to make the paste documentation view-able in the context of a specific version of paste?

**Comment by SiTaggart** [comment, marked as answer]
Hi @eflann 

You got us. You caught us red-handed. We did a whoopsie and had to do a major refactor and hoped no one would notice. The code for the latest docs are now on NPM under Paste Core V17 to install, so you should be good to just version bump and follow the latest documentation.

We know there can be a delay in NPM matching the docs, and it comes up sometimes if we're not quick enough. We've toyed with the idea of a development branch which continuously deploys the docs to a different sub-domain, and the main docs domain is only ever updated after a versioning event being merged to `main`. It happens so infrequently though it's hard to prioritize.

The docs for past versions are now listed on the upgrade guide for each major release, as an example the link to the v16 docs lives here https://paste.twilio.design/core/upgrade-guide#core-16-0-0. That should help some folks who find themselves pinned to old versions longer than they should need to be.
